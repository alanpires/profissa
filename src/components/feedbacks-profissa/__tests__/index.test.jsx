import React from "react";
import FeedbacksProfissa from "../index";
import renderer from "react-test-renderer";

describe("FeedbacksProfissa Test", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FeedbacksProfissa />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
