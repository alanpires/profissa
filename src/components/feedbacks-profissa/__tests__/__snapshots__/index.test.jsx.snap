// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbacksProfissa Test renders correctly 1`] = `
<div
  className="sc-hKFyIo dYSuMn"
>
  <div
    className="sc-eCApGN dfAQhT"
  >
    <h1>
      Feedbacks recebidos
    </h1>
    <div
      className="sc-bdnylx jUGZkC"
    >
      <div
        className="sc-gtssRu eEZLDO"
      >
        <div
          className="sc-dlnjPT hDdslF"
        >
          <img
            src="ellipse4.png"
          />
        </div>
        <p>
          João
        </p>
      </div>
      <span>
        Excelente profissional, fez um ótimo trabalho, recomendo a todos.
      </span>
    </div>
    <div
      className="sc-bdnylx jUGZkC"
    >
      <div
        className="sc-gtssRu eEZLDO"
      >
        <div
          className="sc-dlnjPT hDdslF"
        >
          <img
            src="ellipse5.png"
          />
        </div>
        <p>
          Maria
        </p>
      </div>
      <span>
        Excelente profissional, fez um ótimo trabalho, recomendo a todos.
      </span>
    </div>
    <div
      className="sc-bdnylx jUGZkC"
    >
      <div
        className="sc-gtssRu eEZLDO"
      >
        <div
          className="sc-dlnjPT hDdslF"
        >
          <img
            src="ellipse4.png"
          />
        </div>
        <p>
          Pedro
        </p>
      </div>
      <span>
        Excelente profissional, fez um ótimo trabalho, recomendo a todos.
      </span>
    </div>
    <div
      className="sc-bdnylx jUGZkC"
    >
      <div
        className="sc-gtssRu eEZLDO"
      >
        <div
          className="sc-dlnjPT hDdslF"
        >
          <img
            src="ellipse5.png"
          />
        </div>
        <p>
          Lucas
        </p>
      </div>
      <span>
        Excelente profissional, fez um ótimo trabalho, recomendo a todos.
      </span>
    </div>
  </div>
</div>
`;
